### PR TITLE
Examples: Load cubemap mipmaps with level

### PR DESCRIPTION
**Pull request by liamlangli**
Load cubemap mipmaps with mipmap level.
Make sure each level of cubemap was loaded to the correct mipmap slot.

#17072

### CODE REVIEW

**Comment by Mugen87** [comment]
http://raw.githack.com/AngusLang/three.js/fix/load-cubemap-mipmaps-with-level/examples/webgl_materials_cubemap_mipmaps.html

File: examples/webgl_materials_cubemap_mipmaps.html [hunk 1]
```diff
@@ -28,21 +28,24 @@
...
 				async function loadCubeTexture( urls ) {
 
 					return new Promise( function ( resolve, reject) {
 
-						new THREE.CubeTextureLoader().load( urls, function ( cubeTexture ) {
+						new THREE.CubeTextureLoader().load( urls,function ( cubeTexture ) {
 
 							resolve( cubeTexture );
 
+						}, function ( error ) {
```
Comment (Mugen87) [inline examples/webgl_materials_cubemap_mipmaps.html]: Any reasons for adding an `onError` callback with no implementation?

**Comment by Mugen87** [comment]
The warning itself is actually gone and both spheres always correctly appear.

File: examples/webgl_materials_cubemap_mipmaps.html (hunk 1 above)
Comment (AngusLang) [reply examples/webgl_materials_cubemap_mipmaps.html]: add for debug, forget to remove  : )

**Review by Mugen87 — APPROVED** [review]

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**